docs/pyramid.rst

```diff
@@ -304,6 +304,13 @@ often have dependencies beyond those of the Pyramid core.
 
   - Version Control: https://github.com/ptahproject/ptah
 
+`Ringo <http://ringo-dev.intevation.de>`_
+  Ringo is a extensible high-level web application framework with strength in
```

Comment:
Please correct grammar to this: "Ringo is an extensible". Then it looks good, and I'll merge this PR. Thank you!